main.go

```diff
@@ -206,8 +205,11 @@ func main() { //nolint:funlen,maintidx
 			&configv1.Authentication{}: {
 				Field: fields.Set{"metadata.name": cluster.ClusterAuthenticationObj}.AsSelector(),
 			},
-			// for prometheus and black-box deployment and ones we owns
-			&appsv1.Deployment{}: {Namespaces: deploymentCache},
+			// for deployments in application namespace + operator namespace + monitoring namespace + rhods-notebooks
+			// default wb namespace is not needed since no deployment only sfs,  but our label is on it so we cache it as well
+			&appsv1.Deployment{}: {
+				Label: labels.Set{"opendatahub.io/generated-namespace": "true"}.AsSelector(),
```

Comment:
is "generated-namespace" added to a deployment ?

Reply:
i messed up deployment and namespace